### PR TITLE
Small videolayout refactoring

### PR DESCRIPTION
**Pull request by paweldomas**
The PR does few changes which should not affect functionality, but move some files around. See commit messages for more details.

### CODE REVIEW

File: modules/UI/videolayout/RemoteVideo.js [hunk 1]
```diff
@@ -109,7 +108,7 @@ RemoteVideo.prototype._generatePopupContent = function () {
     // Delegate event to the document.
     $(document).on("click", "#mutelink_" + this.id, function(){
 
-        if (this.isMuted)
+        if (this.isAudioMuted)
             return;
```
Comment (damencho) [inline modules/UI/videolayout/RemoteVideo.js]: Was this a bug, or this is how it is suppose to work?
It was using this.isMuted, which is handled in RemoteVideo by this.isVideoMuted, but here we use this.isAudioMuted?

Comment (paweldomas) [inline modules/UI/videolayout/RemoteVideo.js]: I guess it was a bug

File: modules/UI/videolayout/RemoteVideo.js [hunk 2]
```diff
@@ -153,7 +152,7 @@ RemoteVideo.prototype._generatePopupContent = function () {
  */
 RemoteVideo.prototype.updateRemoteVideoMenu = function (isMuted, force) {
 
-    this.isMuted = isMuted;
+    this.isVideoMuted = isMuted;
```
Comment (damencho) [inline modules/UI/videolayout/RemoteVideo.js]: But we call this method from VideoLayout.onAudioMute, so this is not video muted. 
The isMuted was introduced with commit https://github.com/jitsi/jitsi-meet/commit/5654d34ee88e0ec25fbff78c6ece53cabb832646 so I think we just need to rename isMuted to isAudioMuted.

Comment (paweldomas) [inline modules/UI/videolayout/RemoteVideo.js]: I will have a look tomorrow. The docs are misleading

Comment (paweldomas) [inline modules/UI/videolayout/RemoteVideo.js]: Yes, you're right. Thanks ! I've updated the PR.

**Pull request merged**